js-runner/webpack.common.js

```diff
@@ -49,10 +53,11 @@ const nodeConfig = merge(common, {
   entry: "./src/runner.js",
   output: {
     filename: "grain-runner.js",
-    path: __dirname + "/dist",
```

Comment:
The migration guide recommended removing this if it matched the default (ours does)